projects/ad5766_sdz/Readme.md

```diff
@@ -0,0 +1,8 @@
+# AD5766-SDZ HDL Project
+
+Here are some pointers to help you:
+  * [Board Product Page](https://www.analog.com/EVAL-AD5766)
+  * Parts : [16-Channel, 16-Bit Voltage Output denseDAC](https://www.analog.com/ad5766)
+  * Project Doc: 
+  * HDL Doc: 
+  * Linux Drivers: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/iio/dac/ad5766.c?id=HEAD
```

Comment:
Do we need the extra new line and id=HEAD at the end of the link?

Reply:
Done